doc/releases/changelog-0.42.0.md

```diff
@@ -4,66 +4,31 @@
 
 <h3>New features since last release</h3>
 
-* Leveraging quantum just-in-time compilation to optimize parameterized hybrid workflows with the quantum 
-  natural gradient optimizer is now possible with the new :class:`~.QNGOptimizerQJIT` optimizer. 
-  [(#7452)](https://github.com/PennyLaneAI/pennylane/pull/7452)
-  
-  The :class:`~.QNGOptimizerQJIT` optimizer offers a `jax.jit`- and `qml.qjit`-compatible analogue to the existing 
-  :class:`~.QNGOptimizer` with an Optax-like interface:
-
-  ```python
-  import pennylane as qml
-  import jax.numpy as jnp
-
-  @qml.qjit(autograph=True)
-  def workflow():
-      dev = qml.device("lightning.qubit", wires=2)
-  
-      @qml.qnode(dev)
-      def circuit(params):
-          qml.RX(params[0], wires=0)
-          qml.RY(params[1], wires=1)
-          return qml.expval(qml.Z(0) + qml.X(1))
-  
-      opt = qml.QNGOptimizerQJIT(stepsize=0.2)
-  
-      params = jnp.array([0.1, 0.2])
-      state = opt.init(params)
-      for _ in range(100):
-          params, state = opt.step(circuit, params, state)
-  
-      return params
-  ```
-
-  ```pycon
-  >>> workflow()
-  Array([ 3.14159265, -1.57079633], dtype=float64)
-  ```
-
 <h4>State-of-the-art templates and decompositions 🐝</h4>
 
-* The decompositions of `SingleExcitation`, `SingleExcitationMinus` and `SingleExcitationPlus`
-  have been reduced to fewer rotations and/or (CNOT|CZ|CY) gates. This leads to lower circuit cost
-  when decomposing these gates, both when focusing on two-qubit gates or on non-Clifford gates.
-  [(#7771)](https://github.com/PennyLaneAI/pennylane/pull/7771)
-
-* A new decomposition based on *unary iteration* has been added to :class:`qml.Select`.
-  This decomposition reduces the :class:`T` count significantly, and uses :math:`c-1`
-  auxiliary wires for a :class:`qml.Select` operation with :math:`c` control wires.
-  Unary iteration leverages these auxiliary wires to store intermediate values for reuse
-  among the different multi-controlled operators, avoiding unnecessary recomputation.
-  Check out the documentation for a thorough explanation.
+* A new decomposition using *unary iteration* has been added to :class:`~.Select` based on 
+  [arXiv:1805.03662](https://arxiv.org/pdf/1805.03662). This state-of-the-art decomposition reduces 
```

Comment:
This should help with the error `Duplicate explicit target name: "arxiv:1805.03662".` from docutils

```suggestion
* A new decomposition using [unary iteration](https://arxiv.org/pdf/1805.03662) has been added to :class:`~.Select`.
  This state-of-the-art decomposition reduces 
```